NDB.Covid19/NDB.Covid19.iOS/Views/Settings/SettingsViewController.cs

```diff
@@ -35,7 +36,7 @@ public override void ViewDidLoad()
             _tableSource = new SettingsTableViewSource(_viewModel.SettingItemList);
             CloseBtn.AccessibilityLabel = SettingsViewModel.SETTINGS_ITEM_ACCESSIBILITY_CLOSE_BUTTON;
             fhiLogo.AccessibilityLabel = InfectionStatusViewModel.SMITTESPORING_FHI_LOGO_ACCESSIBILITY;
```

Comment:
Remove this duplicate code if you are setting it now in `SetupAccessibility()`

Reply:
Fixed

Reply:
Fixed